nikola/conf.py.in

```diff
@@ -343,6 +356,25 @@ HIDDEN_TAGS = ['mathjax']
 # However, more obscure tags can be hidden from the tag index page.
 # TAGLIST_MINIMUM_POSTS = 1
 
+# A list of dictionaries specifying tags which translate to each other.
+# For example:
+#   [
+#     {'en': 'car', 'de': 'Auto'},
+#     {'en': 'window', 'fr': 'fenêtre'},
```

Comment:
Since that example is now gone, I added `{'en': 'work', 'fr': 'travail', 'de': 'Arbeit'},` in the post sections example.